client/src/components/DecisionButtons.vue

```diff
@@ -120,6 +120,12 @@ export default {
     newComment() {
       this.warnDecision = false;
     },
+    confirmedPresent() {
+      this.warnDecision = false;
+    },
+    confirmedAbsent() {
+      this.warnDecision = false;
```

Comment:
If all artifacts are cycled back to unconfirmed state, is that case handled?

Reply:
In that case, the length of the `confirmedPresent` and `confirmedAbsent` lists would be 0, so the check at the top of `handleCommentSave` would catch and tell the user to make a comment.